x-pack/plugins/maps/server/mvt/mvt_routes.ts

```diff
@@ -68,6 +69,7 @@ export function initMVTRoutes({
       };
       try {
         tileRequest = getHitsTileRequest({
+          buffer: 'buffer' in query ? (query.buffer as number) : 5,
```

Comment:
Why do we use `parseInt` in line 139, but not here?

Reply:
Thanks for pointing this out. I have switched this to use parseInt as well.